docs/documentation.md

```diff
@@ -79,12 +79,13 @@ Platform versions follow standard numbering, and releases below 1 should be cons
 #### List of versions (newest on top):
 
 - [release 1]()
+- [legacy]() - for simulations with input data generated with [NorESMhub/noresm-lsp-input](https://github.com/NorESMhub/noresm-lsp-input) repo
 - [archive tag](https://github.com/NorESMhub/noresm-land-sites-platform/releases/tag/archive)
 - [tag 0.1.0-dev](https://github.com/NorESMhub/noresm-land-sites-platform/releases/tag/v0.1.0)
 
 #### Model versions
 
-The platform is primarily designed to run the land model (CLM) with the Norwegian Earth System Model. The versions of FATES and CLM therefore have to be in line with stable NorESM versions. NorESM is integrated into the platform via a dedicated `noresm_landsites` branch in the [NorESMhub/NorESM repository](https://github.com/NorESMhub/NorESM/tree/noresm_landsites).
+The platform is primarily designed to run the land model (CLM) with the Norwegian Earth System Model. The versions of FATES and CLM therefore have to be in line with stable NorESM versions. NorESM is integrated into the platform via a dedicated `noresm_landsites` branch in the [NorESMhub/NorESM repository](https://github.com/NorESMhub/NorESM/tree/noresm_landsites). You can see the component model versions in the `Externals.cfg` file on that branch.
```

Comment:
In the coming version, we're using the latest version of NorESM (2.0.5). Currently, develop points to 2.0.2, but I'll update it when my changes are ready.

You can refer to this file for Externals.cfg (maybe after it's merged into main: https://github.com/NorESMhub/noresm-land-sites-platform/blob/develop/resources/overwrites/Externals.cfg.

Reply:
OK, tried to update now.